server/services/diplomacy.ts

```diff
@@ -105,17 +105,15 @@ export default class DiplomacyService extends EventEmitter {
         return diplomaticStatuses;
     }
 
-    getAlliesOfPlayer(game: Game, player: Player): Player[] {
+    getAlliesOfPlayer(game: Game, player: Player, actualOrOffer : Boolean = false): Player[] {
```

Comment:
We need to be very careful here. What calls this function? Will combat be affected by this change?